topics/single-cell/tutorials/scrna-case_monocle3-rstudio/preamble.md

```diff
@@ -94,7 +94,7 @@ You have two options for how to proceed with this JupyterLab tutorial - you can
 >
 > 2. Run 
 >    ```
->    wget {{ ipynbpath }}
+>    download.file("{{ ipynbpath }}", "singlecell.ipynb") 
```

Comment:
`download.file` doesn't match with the above command to "open a terminal in jupyterlab" where `wget` is appropriate.